content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -89,10 +89,9 @@ isolate the users in the container from the users in the node.
 This means containers can run as root and be mapped to a non-root user on the
 host. Inside the container the process will think it is running as root (and
 therefore tools like `apt`, `yum`, etc. work fine), while in reality the process
-doesn't have privileges on the host. You can verify this, for example, if you
-check the user the container process is running `ps` from the host. The user
-`ps` shows is not the same as the user you see if you execute inside the
-container the command `id`.
+doesn't have privileges on the host. You can verify this, for example, if you check
+the user container the process is running `ps` from the host. The user output
```

Comment:
how do you parse "if you check the user container the process is running `ps` from the host"?

What is the objective for the verb "check"? "user container"? Then what is "the process is running `ps` from the host"? 

This sentence is not valid English syntax, right?

Reply:
I also don't understand this sentence.